source/docs/software/driverstation/driver-station-labview-dashboard.rst

```diff
@@ -0,0 +1,127 @@
+FRC Driver Station LabVIEW Dashboard
+====================================
+
+The Dashboard application installed and launched by the FRC Driver Station is a LabVIEW program designed to provide teams with basic feedback from their robot, with the ability to expand and customize the information to suit their needs. This Dashboard application uses Network Tables and contains a variety of tools that teams may find useful.
+
+LabVIEW Dashboard
+-----------------
+
+.. image:: images/driver-station-labview-dashboard/labview-dashboard.png
```

Comment:
Update to picture of 2020 dashboard

Reply:
Updated to:
![image](https://user-images.githubusercontent.com/15172054/81518453-fa760180-92f2-11ea-9956-64ebe8f2ea4c.png)